### ISSUE TITLE
Cannot publish to AMT sandbox, is HTTPS required?

### ISSUE THREAD

**Issue by jeffshih** [cvondrick/vatic#62]
I successfully deployed the tool in off-line version. Now I want to deploy it on AMT Sandbox but I get this error when running `turkic publish`:
```
turkic.api.CommunicationError: The value "http://localhost//?id=13" is not valid for the parameter ExternalURL.
This URL must use the HTTPS protocol.. (1482818198548 s)
```

In my `config.py` file I have this:

```
sandbox     = True # if true, put on workersandbox.mturk.com
localhost   = "http://localhost/"
...
```
Is it required to use HTTPS? I have not seen any reference to it in the instructions. If someone has run into the same problem I would appreciate a workaround solution if available or confirmation that HTTPS is needed.

Best,
Jeff

**Comment by gsikheria** [comment]
Hi @jeffshih 

I'm facing issue while connecting with Amazon Mturk sandbox. My database connection and host connection are fine.
I'm new to Mturk, Can you guide me how to get these configuration

signature = "" # AWS secret access key ?
accesskey = "" # AWS access key ID ?

Thanks in advance

**Comment by jeffshih** [comment]
Hi @gsikheria 

You can find these on the [amazon mturk website ](http://docs.aws.amazon.com/AWSMechTurk/latest/AWSMechanicalTurkGettingStartedGuide/SetUp.html)

Just follow the steps till stage 4, Vatic will do the rest part.

**Comment by PieterFabry** [comment]
Any solutions @jeffshih ? I am stuck at the same place, decided to try to https route but then am getting issues connecting to the appache server iteslf in the trukic status --verify

**Comment by jeffshih** [comment]
@PieterFabry  Nah, I can't solve this problem, so I first use a self assigned SSL certificate for testing and verification of the publishing procedure(You can easily google for some way to get a self-assigned SSL), then I buy an address and a SSL certificate for the official publishing.

**Comment by hocherie** [comment]
Are there any updates on this issue?

I have the same problem where it returns the error,
"The value "http://localhost/?id=5" is not valid for the parameter ExternalURL. This URL must use the HTTPS protocol.."

**Comment by xiao617** [comment]
I encounter same problem too, is there any solution?

**Comment by Sayan-Paul** [comment]
Did anyone find a solution?
